python/nrel/routee/compass/io/generate_dataset.py

```diff
@@ -209,3 +206,38 @@ def replace_id(vertex_uuid):
             with importlib.resources.as_file(model_file) as model_path:
                 model_dst = model_output_directory / model_path.name
                 shutil.copy(model_path, model_dst)
+
+#Function written by Nick Reinicke
+def compass_heading(point1, point2):
```

Comment:
Could we move these functions into the [utils](https://github.com/NREL/routee-compass/blob/main/python/nrel/routee/compass/io/utils.py) module?

Reply:
> Could we move these functions into the [utils](https://github.com/NREL/routee-compass/blob/main/python/nrel/routee/compass/io/utils.py) module?

Moved with commit b9f77261ebed959dd01386be240f4b2ee06d9537